src/Organizations/OrganizationsList/OrganizationsListFilter/OrganizationsListFilter.js

```diff
@@ -35,6 +35,7 @@ const OrganizationsListFilter = ({
     <AccordionSet>
       <AcqCheckboxFilter
         activeFilters={activeFilters[FILTERS.STATUS]}
+        id={`org-filter-${FILTERS.STATUS}`}
```

Comment:
there is no need to use templates here. It would make sense if you specify such template literals inside common components, like ```
id={`org-filter-${name}`}`
```
but since you specify this in every filter using the constant here is kind of extra coupling without clear reason.
However, maybe you know some, please share

Reply:
I just tried to follow ids naming that have already been in `OrganizationsListFilter` for [`AcqUnitFilter`](https://github.com/folio-org/ui-organizations/blob/master/src/Organizations/OrganizationsList/OrganizationsListFilter/OrganizationsListFilter.js#L83)